lightkurve/tests/test_targetpixelfile.py

```diff
@@ -51,6 +55,20 @@ def test_tpf_plot():
     with pytest.raises(ValueError):
         tpf.plot(scale="blabla")
 
+def test_find():
```

Comment:
Rename to `test_get_sources()` to make it explicit which function is being tested.